.github/workflows/preview-html.yml

```diff
@@ -2,34 +2,32 @@ name: Preview HTML on Pull Requests
 on:
   pull_request:
```

Comment:
If you want this to run of PRs from forks, you might need to use `pull_request_target` https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/
Downside is that changes don't run till they land in the default branch